DataGenerator/DataGenerator.py

```diff
@@ -96,7 +95,8 @@ def __getitem__(self, id):
 
 ### DataLoader
 class DataModule(LightningDataModule):
-    def __init__(self, mastersheet, label, config, keys, train_transform = None, val_transform = None, batch_size = 64, **kwargs):
+    def __init__(self, mastersheet, label, config, keys, train_transform = None, val_transform = None, batch_size = 64,
```

Comment:
Since they are not used in this function you don't need to add numerical_norm. They are pacakged with the **kwargs and just passed along with everything else. Please name only those that are directly used in this function.